base_delivery_carrier_extensions/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# © 2020 FactorLibre - Zahra Velasco <zahra.velasco@factorlibre.com>
```

Comment:
```suggestion
# Copyright 2020 FactorLibre - Zahra Velasco <zahra.velasco@factorlibre.com>
```
It is stated in copyright laws that the whole word is needed, so we have changed this all around